.github/workflows/build-dotnet.yml

```
@@ -16,8 +16,7 @@ on:
         required: false
         type: string
         default: 6.0.*
-        description: >
-          Version of the .NET SDK to set up. Defaults to 6.0.*.
+        description: Version of the .NET SDK to set up. Defaults to "6.0.*".
```

Would it be exaggerated to reuse the default value in the description? Or is this even possible? I'm still not deep enough into GitHub Actions to know. Do you?

```suggestion
        description: Version of the .NET SDK to set up. Defaults to "${{ inputs.dotnet-version.default }}".
```

---

OK, this is probably overkill. 🤭

---

Does it work? Does it look revolting? Also yes. Also why do we even include "Defaults to X" in the description when you can see it right above? As far as I know, there is no GUI for inspecting or importing workflows so there is no benefit.

---

Just because we can, doesn't mean we should.

I agree with dropping the "Defaults to XYZ" everywhere.